FOO.md

```diff
@@ -0,0 +1 @@
+this is FOO.md
```

Comment:
This is my *outdated* comment, created via [this API](https://developer.github.com/v3/pulls/comments/#create-a-comment).